internal/controller/controllers/agentserviceconfig_controller_test.go

```diff
@@ -440,6 +506,14 @@ func newASCWithCMAnnotation() *aiv1beta1.AgentServiceConfig {
 	return asc
 }
 
+// func newASCWithMirrorRegistryConfig() *aiv1beta1.AgentServiceConfig {
```

Comment:
Can we remove the commented code or at least add a note why it's worth keeping it?

Reply:
> Can we remove the commented code ~or at least add a note why it's worth keeping it~?

:rofl:

Reply:
Yeah. I saw that last night but was certain there would be an opportunity to fix.

Reply:
Ended up needing it.